dev/MRTCore/mrt/mrm/include/mrm/MrmEnvironment.h

```diff
@@ -6,43 +6,43 @@
 namespace Microsoft::Resources
 {
 
-/*! 
+/*!
  * Describes a single resource qualifier.
  */
 struct ResourceQualifier
 {
-    /*! 
```

Comment:
(Are these whitespace changes intentional?  It makes reviewing a bit difficult...)

Reply:
It is in my visual studio code settings to trim whitespace when saving a file.

![image](https://github.com/microsoft/WindowsAppSDK/assets/48363984/d7bfe301-ec53-4531-a869-48b69e365a89)

You can turn off showing whitespace here.